### ISSUE TITLE
Test Batchsize

### ISSUE THREAD

**Issue by cocoxw** [DeokyunKim/Progressive-Face-Super-Resolution#25]
Thank you for sharing! Recently, I encountered a problem during the test. When the test batchsize changes, the final results will be a little different. Could you give me some advice, thanks. ^_^

**Comment by cocoxw** [comment]
The final result means the average PSNR generated by eval.py

**Comment by DeokyunKim** [comment]
Hi, cocoxw.
Thank you for your interest in our work.

  Your question is that the PSNR values are different along with batch size. The PSNR values are slightly different since the batch norm layer of our face SR model calculates the mean and variance of each batch. To resolve the problem,  there are some options such as using another normalization layer or removing batch norm layers.
 
   We note that the uploaded trained model is re-trained best version model through extensive experiments.

If you have more questions, please comment here.

Best regards,
Deokyun Kim

**Comment by cocoxw** [comment]
I got it. Thank you for your reply.
